config/config_exec/credentials.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+"browserstack":{
+        "username":"vishal538",
+        "key":"NqQyPLgqydEAegmxE5uh",
```

Comment:
please remove the access key

Reply:
Removed it, Need to discuss more about this , as you suggested provide this  by environment variable but same issue with windows and linux command line